packages/conf-rust-llvm/conf-rust-llvm.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "Léo Andrès <contact@ndrs.fr>"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+build: [
+  ["which" "rust-lld"] { os-distribution != "homebrew" }
```

Comment:
Same here, it's a pain to test on homebrew... but I believe it's good enough as is.

Reply:
```suggestion
  ["which" "rust-lld"]
```

Reply:
This is going to fail IIRC

Reply:
It'sthe same issue as: https://github.com/ocaml/opam-repository/pull/27381/files#r1954570873